From 39315f9f8dd5a16b4561c5efffc6114c75835011 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 22 May 2015 01:37:16 +0200
Subject: [PATCH] NEWS: fix date

Ah, bummer, it's tagged an now I realize I didn't fix the date. Let's do
so now, post-commit.
---
 NEWS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/NEWS b/NEWS
index 7a4621cc5d..f72f502129 100644
--- a/NEWS
+++ b/NEWS
@@ -227,7 +227,7 @@ CHANGES WITH 220:
         Gundersen, Torstein Husebø, Umut Tezduyar Lindskog, Will
         Woods, Zachary Cook, Zbigniew Jędrzejewski-Szmek
 
-        -- Berlin, 2015-05-??
+        -- Berlin, 2015-05-22
 
 CHANGES WITH 219:
 
